locales/ne-NP.ftl

```diff
@@ -0,0 +1,20 @@
+title = Firefox Quantum Release क्यम्पैन
+project-name = Firefox Quantum Release
+main-title = Firefox को पलटवार
+main-description = नयाँ Firefox को बारेमा मान्छेहरुलाई भन्नुहोस। 14 नवम्बर को सांझ साढ़े-सात बजे आउनुहोस अनि नयाँ Firefox को बारेमा आफ्नो मनमा भएको राय ट्वीट गर्नुहोस, साथै #NewFirefox जोड्न चही नबिर्सनु होला। तपाईं Firefox को ट्वीटस रिट्वीट पनि गर्नसक्नु हुन्छ ।
+thunderclap-cta = Firefox Quantum लाई सहयोग गर्नुहोस 
+thunderclap-link = https://www.thunderclap.it/projects/64595-firefox-quantum-launch-day
+official-tweet-cta = आधिकारिक ट्वीट लाई रीट्वीट गर्नुहोस 
+tweet-1 = आयो आयो अब #NewFirefox, पहिले भन्दा दुइ गुना छिटो । 💥 अहिले नै Download गर्नुहोस: https://mzl.la/newfirefox
+tweet-2 = #NewFirefox __________ भन्दा पनि छिटो छ? डाउनलोड गर्नुहोस नया #Firefox अनि अहिले थाहापाउनुहोस: https://mzl.la/newfirefox । धन्यबाद @Firefox!
+tweet-3 = #NewFirefox आइसक्यो, अब पहिला भन्दा धेरै सक्षम, राम्रो र छिटो। कसरी? #NewFirefox 🦊 Download गर्नुस् अनि अफै थाहा पाउनु हुनेछ:  https://mzl.la/newfirefox
+own-tweet-cta = आफ्नै शब्दमा ट्वीट गर्नुहोस 
+hashtag = #newfirefox
+github = GitHub
+legal = कानून
+license = अनुज्ञप्ति
+guidelines = दिशा निर्देश
+mozilla-description =
+  Mozilla एक नॉन-प्रॉफिट कंपनी हो जो जुटिरहेको छ तपाईंको online experience को रिमोट-कंट्रोल तपाईंलाई दिनको लागि र वेबको नक्सा मन्छेको हितको लागि परिवर्तन गर्न। बिस्तृत रुपमा जन्नकोलागि:
+  <a href="https://mozilla.org" target="_blank">mozilla.org</a>
+official-tweet-id = 930435170288656384
```

Comment:
Could you please update the ID to include the latest from the en-US file?

Reply:
Done 😄